ansible/roles/opensearch/templates/opensearch-security/backup.sh.j2

```diff
@@ -16,7 +16,8 @@ docker exec {{ opensearch_docker_container }} \
     -icl -nhnv -cd /tmp/backup -r
 
 # Compress backup directory
-tar -cvzf ${BACKUP_DIR}/${BACKUP_NAME} -C "{{ opensearch_security_backup_dir }}" .
+cd "{{ opensearch_security_backup_dir }}"
+tar -cvzf ${BACKUP_DIR}/${BACKUP_NAME} *.yml
```

Comment:
Doesn't it work if you keep the -C and add the `*.yml`?

Reply:
No, it returns this error

```
$ tar -cvzf backup.tgz -C /tmp/backup/ "*.yml"
tar: *.yml: Cannot stat: No such file or directory
```